tests/unittests/BackendTestUtils.cpp

```diff
@@ -220,6 +220,40 @@ void dispatchInference(const std::string &fname,
   contexts[0].release();
 }
 
+/// Helper that iterates over all of the Placeholders in \p PHs and converts the
+/// Tensor pair found in \p bindings to the same type as the Placeholder if
+/// necessary.
+static void convertBindingsToCorrectType(PlaceholderBindings &bindings,
```

Comment:
Does a seperate PH list need to be passed in here?  We could iterate over all PHs in the bindings.

Reply:
@gcatron Hm so after reading it again, I think I did it like this because I wanted to be positive I only changed the placeholders that came from the specific Function I was compiling (so I pass in `Function::findPlaceholders()` to this function). I can't think of a specific case where we wouldn't want to fix any other mismatches but I guess since it felt like an aggressive thing to do to convert Tensors I was being conservative. I think I'm fine with changing it to convert all of the bindings but no strong feelings, WDYT?

Reply:
I think it's fine as is then. Being more specific also allows for more specific testing too.